src/OpenURLServer.js

```diff
@@ -41,6 +38,24 @@ class OpenURLServer {
       });
     }
 
+    const svcCfg = cfg.getValues()?.services?.[symbol] ?? cfg.getValues();
+    if (svcCfg?.reqIdHeader) {
+      let fromHeader = ctx.req.headers?.[svcCfg?.reqIdHeader];
+      if (typeof fromHeader === 'string') {
+        if (svcCfg?.reqIdToUpper) fromHeader = fromHeader.toUpperCase();
```

Comment:
All these `?.` make me itchy. We know `svcCfg` is defined, otherwise this clause's guard condition `svcCfg?.reqIdHeader` could not have been true, so we should just use `.` here.